### PR TITLE
Update for the Seek Thermal Camera

### PR DESCRIPTION
**Pull request by Ediolot**
Update for the configuration of the new Seek Thermal Camera module. Now works on windows and can be configured to use a flat and bias images for calibration, as well as temperature values.

The documentation has been updated and we included some example binary files with examples of the flat and bias images.

### CODE REVIEW

File: src/Documentation/UserManual/DeviceInfraredSeekCam.dox [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
 
 \image html DeviceInfraredSeekCam.png
 
-This device only works in Linux. We hope in the next months to integrate in Windows. 
-
 \section InfraredSeekCamConfigInstallation
 
-To use the camera you have to grant privileges to write to the camera. We have made a small script that you hace to execute with root privileges and you have to execute only just one time.
-You can found the instructions in:
+To use the camera you have to grant privileges to write to the camera. For Linux, we have made a small script that you hace to execute with root privileges and you have to execute only just one time. For Windows, the WinUSB driver is needed.
```
Comment (adamrankin) [inline src/Documentation/UserManual/DeviceInfraredSeekCam.dox]: Please provide a couple of details about what the script does. Root access scripts are a risk to users, so we might as well provide some info.

File: src/PlusDataCollection/InfraredSeekCam/vtkInfraredSeekCam.h [hunk 2]
```diff
@@ -47,6 +47,10 @@ class vtkPlusDataCollectionExport vtkInfraredSeekCam : public vtkPlusDevice
   /*! Verify the device is correctly configured */
   virtual PlusStatus NotifyConfigured();
 
+  bool readBinaryFile(const std::string& filename, cv::Mat& temp) const;
```
Comment (adamrankin) [inline src/PlusDataCollection/InfraredSeekCam/vtkInfraredSeekCam.h]: Minor changes, but could you apply the code formatting guidelines?

readBinaryFile -> ReadBinaryFile
readImage -> ReadImage
Settemp -> SetTemp

File: src/PlusDataCollection/InfraredSeekCam/vtkInfraredSeekCam.h [hunk 3]
```diff
@@ -55,8 +59,18 @@ class vtkPlusDataCollectionExport vtkInfraredSeekCam : public vtkPlusDevice
...
-  std::shared_ptr<LibSeek::SeekThermalPro>         Capture;
-  std::shared_ptr<cv::Mat>                  Frame;
+  std::shared_ptr<LibSeek::SeekThermalPro> Capture;
+  std::shared_ptr<cv::Mat>                 Frame;
+
+  cv::Mat Flat;
+  cv::Mat Bias;
+  cv::Mat FrameFloat;    // Output frame in float precision
+  cv::Mat FrameInt;      // Input frame in integer precision
+  bool existsFlat;       // Checks if the flat image could be loaded
```
Comment (adamrankin) [inline src/PlusDataCollection/InfraredSeekCam/vtkInfraredSeekCam.h]: Member variable style:
existsFlat -> ExistsFlat
(etc...)

File: src/PlusDataCollection/InfraredSeekCam/vtkInfraredSeekCam.cxx [hunk 4]
```diff
@@ -158,3 +213,31 @@ PlusStatus vtkInfraredSeekCam::NotifyConfigured()
 
   return PLUS_SUCCESS;
 }
+
```
Comment (adamrankin) [inline src/PlusDataCollection/InfraredSeekCam/vtkInfraredSeekCam.cxx]: Style:
Add 
//-----------------------------------------------------------------------------

before functions

**Comment by Ediolot** [comment]
Updated the documentation and the coding style in the last two commits. The function Settemp was not implemented and was not being used, its declaration was removed.

**Pull request merged**